batchflow/models/torch/base.py

```diff
@@ -999,10 +999,16 @@ def train(self, *args, feed_dict=None, fetches=None, use_lock=True, profile=Fals
 
             # Parse outputs to a desired structure
             if fetches:
```

Comment:
what we really need to make any changes in this code is a list of possible output contents  and a test suite that covers it.

Reply:
Agree. But this requires the code to work, so these changes were made. Before this, saving multiple fetches to different destinations did not work